azure/vnet/main.tf

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# CAF resource
+# ----------------------------------------------------------------------------------------------------------------------
+
+module "azure_region" {
```

Comment:
Out of curiosity, why are we using this module for determining the region?  I am looking at the source code for the module and while it will work, it seems to only be mapping different region naming schemes together (i.e. us-west = "West US" ) and mapping pairings together.  While it is totally fine to use an external module for this we should think about if we want to be reliant on an external module

Reply:
@TysonBeck  stated,
I have no issue with continuing to use that module